### DISCUSSION TITLE
Is plane support CockroachDB ?

### DISCUSSION THREAD

**Discussion by calvin2021y** [jamsocket/plane#825] [Q&A]
CockroachDBprovide more reliable usage compare to Postgres HA cluster, and  compatible with the PostgreSQL dialect.

Is any one try run plane with it?  

Is there a plan to provide office CockroachDB support?

**Comment by paulgb** [comment]
If it's fully Postgres compatible, it _should_ work, although I don't know if Cockroach trades performance for resilience in places that might make it less performant.
